src/transformers/utils/quantization_config.py

```diff
@@ -14,11 +14,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import copy
+import json
+import os
 from dataclasses import dataclass
+from typing import Any, Dict, Union
+
+from ..utils import logging
+
+
+logger = logging.get_logger(__name__)
 
 
 @dataclass
-class BitsAndBytesConfig:
+class BitsAndBytesConfig(object):
```

Comment:
```suggestion
class BitsAndBytesConfig:
```
No need to add this.